src/app/icd/ICDManager.cpp

```diff
@@ -108,14 +115,14 @@ void ICDManager::UpdateOperationState(OperationalState state)
         DeviceLayer::SystemLayer().StartTimer(System::Clock::Timeout(idleModeInterval), OnIdleModeDone, this);
 
         System::Clock::Milliseconds32 slowPollInterval = GetSlowPollingInterval();
-        // When in SIT mode, the slow poll interval SHOULDN'T be greater than the SIT mode polling threshold, per spec.
-        if (mICDMode == ICDMode::SIT && slowPollInterval > GetSITPollingThreshold())
+
+#if 0 // TODO ICD Spec to define this conformance as a SHALL
```

Comment:
I do not understand this TODO. Is this uncomment when Spec is updated? Maybe an action item phrasing would help (explain when this should be uncommented and when it should not be). May even help having a `#define` instead directly if 0.

Reply:
I cannot enforce the max slowPollInterval in SIT mode has the spec mentions "should" rather than "shall" based on a few discussions, this should be changed in the spec. Once the spec is changed, we can enforce it here.

I guess I can use a definition to enable/disable this until then but it might be just temporary.

Reply:
added a define